<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BaseDataBoolCalcWdg</class>
 <widget class="QWidget" name="BaseDataBoolCalcWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hLayoutType">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbType">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>计算类型:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbType">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerType">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="XMatHLine" name="hLine2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutV1">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbV1">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>布尔计算值1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbV1">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbB1">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutV2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbV2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>布尔计算值1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbV2">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbB2">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="XMatHLine" name="hLine1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutRet">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="XMatLabel" name="lbRet">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>计算结果:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatComboBox" name="cmbRet">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerRet">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XMatComboBox</class>
   <extends>QComboBox</extends>
   <header>XMatComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>XMatLabel</class>
   <extends>QLabel</extends>
   <header>XMatLabel.h</header>
  </customwidget>
  <customwidget>
   <class>XMatHLine</class>
   <extends>QWidget</extends>
   <header>XMatHLine.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
